diff --git a/config/ios/ios_sdk.gni b/config/ios/ios_sdk.gni
index 3f0577054..9b871dae0 100644
--- a/config/ios/ios_sdk.gni
+++ b/config/ios/ios_sdk.gni
@@ -32,7 +32,7 @@ declare_args() {
   # ios_enable_code_signing_flag=<bool> flag to make the invocation clearer.
   ios_enable_code_signing = true
   ios_code_signing_identity = ""
-  ios_code_signing_identity_description = "iPhone Developer"
+  ios_code_signing_identity_description = "Apple Development"
 
   # Prefix for CFBundleIdentifier property of iOS bundles (correspond to the
   # "Organization Identifier" in Xcode). Code signing will fail if no mobile
